source/configuration/modules/omelasticsearch.rst

```diff
@@ -80,6 +80,20 @@ Action Parameters
    `rsyslog template <http://www.rsyslog.com/doc/rsyslog_conf_templates.html>`_
    for searchType_, instead of a static string.
 
+.. _pipelineName:
+
+-  **pipelineName**
+   The `ingest node <https://www.elastic.co/guide/en/elasticsearch/reference/5.5/ingest.html>`_ 
```

Comment:
I would use https://www.elastic.co/guide/en/elasticsearch/reference/master/ingest.html instead

Reply:
You are right. But maybe current is better then master: 
https://www.elastic.co/guide/en/elasticsearch/reference/current/ingest.html